recipes/gmm/LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+Getfem++  is  free software;  you  can  redistribute  it  and/or modify it
```

Comment:
Why is this file needed here? Isn't it in the source tarball?

Reply:
there is a "COPYING" file, if I can use that instead let me know

Reply:
I think I can per the conda forge docs, so I'll try that now and if the linters complain I'll undo it